02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -437,27 +452,34 @@ Let's look for the gene set with the highest negative NES.
 
 ```{r}
 gsea_result_df %>%
-  # The `dplyr::arrange()` function sorts the values in ascending order
-  dplyr::arrange(NES)
+  # Use `top_n()` to return the top n observation using `NES` as the ordering
+  # variable -- note the use of `dplyr::desc()` to sort the values in descending order
+  dplyr::top_n(n = 1, wt = dplyr::desc(NES))
```

Comment:
You can use -1 for this purpose instead of `desc()` That's what the docs show: https://dplyr.tidyverse.org/reference/top_n.html